docs/rules/no-unknown-property.md

```diff
@@ -22,6 +22,16 @@ var React = require('react');
 var Hello = <div className="hello">Hello World</div>;
 ```
 
+## Rule Options
+
+HTML properties and attributes which would normally be considered warnings can be whitelisted. If you wanted to allow use of 'class' and 'for' attributes because your Babel configuration handles transforming them to DOM attributes, the configuration would be:
+
+```js
+"rules": {
+  "react/no-unknown-property": [1, { "allow": ["class", "for"] }],
```

Comment:
React doesn't support it, but you can use Babel plugins to handle usage of 'class' and 'for' at transpile time for teams who prefer to keep using them, e.g: https://github.com/insin/babel-plugin-react-html-attrs/issues/4#issuecomment-225307332